### DISCUSSION TITLE
Uninstall Dogecoin Core steps for MAC

### DISCUSSION THREAD

**Discussion by normgithub** [dogecoin/dogecoin#2272] [Q&A] [answered]
Need to reinstall Dogecoin Core to another computer since my MAC ran out of space after installing Dogecoin core which used close to 100GB of space.

Need documentation to uninstalling all related files including removing the wallet.

Thanks.

**Comment by rnicoll** [comment, marked as answer]
There's a data directory in `~/Library/Application Support/Dogecoin/`, but... obviously back up the wallet.dat file from there _just in case_ first. In fact if you're installing on another system, you probably want to copy that `wallet.dat` file over.
